oe_contact_forms.module

```diff
@@ -215,8 +216,15 @@ function _oe_contact_forms_clear_block_cached_definitions(): void {
  */
 function oe_contact_forms_contact_form_presave(EntityInterface $entity) {
   _oe_contact_forms_clear_block_cached_definitions();
-  // Invalidate cache tags if config is changed.
-  Cache::invalidateTags($entity->getCacheTags());
+  // Invalidate form display cache tags if config is changed,
+  // since the EntityFormDisplay is not in storage we have to instantiate here.
+  $entity_form_display = new EntityFormDisplay([
+    'targetEntityType' => 'contact_message',
+    'bundle' => $entity->id(),
+    'mode' => 'corporate_default',
+  ], 'entity_form_display');
+
+  Cache::invalidateTags($entity_form_display->getCacheTags());
```

Comment:
$entity->getCacheTags() should not be clear also?

Reply:
My tests where inconclusive so yes, better to be on the safe side

Reply:
No need, that gets invalidated on save..